Bottom: 021491e87d7a4a576c6d2f7f97025d6ef1155a41
Top:    9d5e87ec3982847e89b8102eccac7589d928ca56
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:36:19 +0000

w32: Add recipe for freetype

(Probably not the latest available, but one I successfully
used with gEDA and PCB before)


---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 36e45d9..04aec82 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -9,6 +9,7 @@ jpeg
 zlib
 libpng
 tiff
+freetype
 pixman
 libiconv
 gettext
diff --git a/w32/recipes/freetype.recipe b/w32/recipes/freetype.recipe
new file mode 100644
index 0000000..1806541
--- /dev/null
+++ b/w32/recipes/freetype.recipe
@@ -0,0 +1,4 @@
+name=freetype
+version=2.4.4
+release=1
+origin=sf
